lib/entity/fields/entity-boolean-field.ts

```diff
@@ -19,8 +19,8 @@ class EntityBooleanField extends EntityField {
     }
   }
 
-  protected valdiateValue(value: EntityValue) {
-    super.valdiateValue(value);
+  protected validateValue(value: EntityValue) {
```

Comment:
I don't always make typos, but when I do they reveal that I copy and paste too often.

Reply:
At least it was consistent! :)